spk/hassio/Makefile

```
@@ -0,0 +1,44 @@
+SPK_NAME = hassio
+SPK_VERS = 20191224
+SPK_REV = 1 
```

Testing, confirmed on `d67e38b` the trailing space on this line breaks build; instead of:
```
(cd /spksrc/spk/hassio/work-avoton-6.1 && tar cpf /spksrc/spk/hassio/../../packages/hassio_avoton-6.1_20191224-1.spk --group=root --owner=root package.tgz INFO scripts conf PACKAGE_ICON.PNG PACKAGE_ICON_256.PNG WIZARD_UIFILES)
```
... you have:
```
(cd /spksrc/spk/hassio/work && tar cpf /spksrc/spk/hassio/../../packages/hassio_noarch-all_20191224-1 --group=root --owner=root package.tgz INFO scripts conf PACKAGE_ICON.PNG PACKAGE_ICON_256.PNG WIZARD_UIFILES)
(cd /spksrc/spk/hassio/work && tar cpf .spk --group=root --owner=root package.tgz INFO scripts conf PACKAGE_ICON.PNG PACKAGE_ICON_256.PNG WIZARD_UIFILES)
```

Need to fix this

---

Perhaps the build process should handle spaces..

```suggestion
SPK_REV = 1
```